### ISSUE TITLE
illigael path with AssemblyFile

### ISSUE THREAD

**Issue by koiszzz** [emerbrito/ILMerge-MSBuild-Task#35]
I install the package in vs 2019 . When installed , then console shows 
```
C:\Users\kois\source\repos\IdCard\IdCard.csproj : error  : 元素 <UsingTask> 中“AssemblyFile”特性的值“>C:\Users\kois\.nuget\packages\msbuild.ilmerge.task\1.1.1\build\..\tools\MSBuild.ILMerge.Task.dll”无效。路径中具有非法字符。  C:\Users\kois\.nuget\packages\msbuild.ilmerge.task\1.1.1\build\MSBuild.ILMerge.Task.targets
```

**Comment by brcaswell** [comment]
@emerbrito are you publishing packages without updating this codebase?

the package I have is 1.0.7 on VS2017, and here we have a reference to version 1.1.1 on 2019.

**Comment by brcaswell** [comment]
actually, I can see from code releases that version 1.0.7 was the latest release https://github.com/emerbrito/ILMerge-MSBuild-Task/releases 

I don't know where this v1.1.1 is coming from @koiszzz .. you may be getting a modified version from a different feed...?

**Comment by koiszzz** [comment]
@brcaswell , Thranks. I've downloaded wrong package <MSBuild.ILMerge.Task>. It was published v1.1.1 at that time.
